inkcut/core/svg.py

```diff
@@ -263,7 +263,7 @@ def parse(self, e):
         else: 
             if rx == 0:
                 rx = ry
-            else:
+            elif ry == 0:
```

Comment:
True, perhaps `None` should be used instead of using `0` when the attribute is omitted which would remove uncertainty?

I actually didn't run into this problem with a graphic, I was porting the code to work with a 3D library and noticed that it didn't look right.